apps/site/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,63 @@
+import Image from "next/image";
+
+import styles from "./Footer.module.scss";
+
+import Mail from "@/assets/icons/mail.svg";
+import Facebook from "@/assets/icons/facebook.svg";
+import Instagram from "@/assets/icons/insta.svg";
+import Twitter from "@/assets/icons/twitter.svg";
+
+const SOCIALS = [
+	{
+		icon: Mail,
+		link: "mailto:hack@uci.edu",
+		alt: "Mail",
+		width: 78,
+		height: 78,
+		marginRight: "25px",
+	},
+	{
+		icon: Facebook,
+		link: "https://www.facebook.com/UCI.Hack/",
+		alt: "Facebook",
+		width: 59,
+		height: 59,
+		marginRight: "16px",
+	},
+	{
+		icon: Instagram,
+		link: "https://www.instagram.com/hackatuci/",
+		alt: "Instagram",
+		width: 91,
+		height: 91,
+		marginRight: "16px",
+	},
+	{
+		icon: Twitter,
+		link: "https://www.twitter.com/HackUCI",
+		alt: "Twitter",
+		width: 64,
+		height: 53,
+		marginRight: "0px",
+	},
+];
+
+export default function Footer() {
+	return (
+		<div className={styles.footer}>
+			<p className={styles.text}>Made with &lt;3 in Irvine, CA - Hack @ UCI</p>
+			<div className={styles.socials}>
+				{SOCIALS.map(({ icon, link, alt, width, height, marginRight }) => (
+					<a
+						key={link}
+						href={link}
+						target="_blank"
+						style={{ marginRight: marginRight }}
```

Comment:
Thought: manually specifying margin sizes seems impractical. Have you considered using a [flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_flexible_box_layout) or [grid layout](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_grid_layout) approach?
Additional note: if we do keep this approach, property names can be specified as shorthand, i.e. `{ marginRight }`

Reply:
My concern with the flexbox approach is that the SVGs from Figma are different sizes, and there are a different amount of space between them according to the Figma